e2e/nomostest/clusters/gke.go

```diff
@@ -68,12 +70,19 @@ func deleteGKECluster(t testing.NTB, name string) error {
 		args = append(args, "--region", *e2e.GCPRegion)
 	}
 	t.Logf("gcloud %s", strings.Join(args, " "))
-	cmd := exec.Command("gcloud", args...)
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return errors.Errorf("failed to delete cluster %s: %v\nstdout/stderr:\n%s", name, err, string(out))
-	}
-	return nil
+	// Sometimes an operation may be happening at the time the deletion request is
```

Comment:
Can we list cluster operations and wait until they're done instead?

Reply:
I think we would need to wrap that in a retry anyways, as other operations could begin after the initial list

Reply:
yes, but then we wouldn't be spamming deletes when we know they won't work.

Reply:
I'm not super opposed to implementing this, but doing this client side feels like it's just overcomplicating things. If there's an in progress operation then the backend can handle it and will give us an error.

The retry has a backoff strategy so this shouldn't be too spammy, although we could make it so that the backoff is configurable and use a more conservative backoff on this retry.